jcl/src/java.base/share/classes/java/lang/ClassLoader.java

```diff
@@ -352,44 +368,31 @@ protected ClassLoader(ClassLoader parentLoader) {
  *@since 9
  */
 protected ClassLoader(String classLoaderName, ClassLoader parentLoader) {
+	this(checkSecurityPermission(), classLoaderName, parentLoader);
+}
+/*[ENDIF]*/
+
+private ClassLoader(Void staticMethodHolder,
+/*[IF Sidecar19-SE]*/
+	String classLoaderName,
```

Comment:
Since this is a private constructor, can we remove the preprocessor tests and just ignore the `classLoaderName` in JDK8?  It would remove the preprocessor checks in the other constructors as well.

Reply:
Agreed, passing `null` to the private `constructor` can avoid `JPP` decoration since `classLoaderName` already is ignored by `Java 8` via existing `JPP`, now the code looks cleaner, thanks. Verified that `pConfig` still compiles. 
Please have another look.